flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/failover/ExponentialDelayRestartBackoffTimeStrategyTest.java

```diff
@@ -36,35 +36,37 @@ class ExponentialDelayRestartBackoffTimeStrategyTest {
     private final Exception failure = new Exception();
 
     @Test
-    void testAlwaysRestart() throws Exception {
+    void testMaxAttempts() {
+        int maxAttempts = 13;
         final ExponentialDelayRestartBackoffTimeStrategy restartStrategy =
                 new ExponentialDelayRestartBackoffTimeStrategy(
-                        new ManualClock(), 1L, 3L, 2.0, 4L, 0.25);
+                        new ManualClock(), 1L, 3L, 2.0, 4L, 0.25, maxAttempts);
```

Comment:
The default value of `ConfigOption` `restart-strategy.exponential-delay.backoff-multiplier` has been set to `1.2`. Is it necessary to update backoffMultiplier to `1.2`?